### DISCUSSION TITLE
Project Portfolio Website

### DISCUSSION THREAD

**Discussion by ajdeve** [thuliteio/doks#967] [Showcase]
I used doks for my personal resume and project portfolio website. Its minimal design and layouts help display projects step by step in a clean manner. It can definitely be used for a personal project portfolio theme. 

**homepage:** https://ajdeve.github.io/
**project layout:** https://ajdeve.github.io/docs/project2/introduction/

Example of project display using documentation feature of doks. 
![image](https://user-images.githubusercontent.com/73873972/209453593-82620981-984b-495e-9ca3-5c8a235a8df3.png)

**Comment by h-enk** [comment]
Very nice website and interesting to see that Doks also could be used for a personal resume and project portfolio website!

Are you ok with me adding your website to the showcase? I think it would inspire others to see what's possible with Doks.

**Reply by ajdeve** [reply]
Of course! Feel free to add it to the showcase. I got much inspiration from featured websites from doks. Thanks for this amazing product.
